doc/docs/modules/ROOT/pages/gds.adoc

```diff
@@ -14,18 +14,17 @@ GDS algorithms are bucketed into five groups:
 
 == GDS operates via Cypher
 
-All of the link:{url-neo4j-gds-manual}[functionality of GDS] is used by issuing Cypher queries.  As such, it is easily
+All of the link:{url-neo4j-gds-manual}[functionality of GDS^] is used by issuing Cypher queries.  As such, it is easily
 accessible via Spark, because the Neo4j Connector for Apache Spark can issue Cypher queries and read their results back.  This combination means
 that you can use Neo4j and GDS as a graph co-processor in an existing ML workflow that you may implement in Apache Spark.
 
 == Example
 
-In the link:{url-gh-spark-notebooks}[sample Zeppelin Notebook repository], there is a GDS example that can be run against
-a Neo4j Sandbox, showing how to use the two together.
+In the link:{url-gh-spark-notebooks}[sample Zeppelin Notebook repository^], there is a GDS example that can be run against a Neo4j Sandbox, showing how to use the two together.
 
 === Create a virtual graph in GDS using Spark
 
-This is very simple, straightforward code; it constructs the right Cypher statement to link:https://neo4j.com/docs/graph-data-science/current/common-usage/creating-graphs/[create a virtual graph in GDS] and returns the results.
+This is very simple, straightforward code; it constructs the right Cypher statement to link:https://neo4j.com/docs/graph-data-science/current/common-usage/projecting-graphs/[create a virtual graph in GDS^] and returns the results.
```

Comment:
Andrea, please, take a look at this part. GDS team has changed a command in 2.0 version, instead of `CALL gds.graph.create` you can find `gds.graph.project`. Does it affect the Spark connector? Maybe you need to change code block.

Reply:
Thank you much for pointing this out, we don't have any direct dependency to the GDS so this change does not affect the Spark Connector